### ISSUE TITLE
textbox vertical alignment

### ISSUE THREAD

**Issue by myrecs** [PHPOffice/PHPWord#1160]
Is it possible somehow to vertical align content of textbox?

**Comment by FBnil** [comment]
`setInnerMarginTop`?

**Comment by myrecs** [comment]
text content inside TextBox can be different height

**Comment by myrecs** [comment]
no, not working. do you test your suggestions before you write?)

**Comment by FBnil** [comment]
No, I just grep from the examples. Source code does not provide "Horizontal Alignment Type", while Vertical Alignment does exist. Did you try in MSWord to align it, then save the document as docx, rename to zip, extract the document.xml, and see if the document.xml has any markup that reveals Horizontal alignment markup?

**Comment by myrecs** [comment]
i think that the key difference is in "v-text-anchor:middle" when vertically  center and that does the job. Is it possible to add this property to library?

**Comment by FBnil** [comment]
v-text-anchor possible values:
top
middle
bottom
top-center
middle-center
bottom-center
top-baseline
bottom-baseline
top-center-baseline
bottom-center-baseline

Documentation states:
`<v:shape..><v:textbox style="v-text-anchor:bottom" ...></v:textbox/></v:shape>`

(not yet) written in 
`./phpoffice/phpword/src/PhpWord/Writer/Word2007/Element/TextBox.php` where it says:
`$xmlWriter->startElement('v:textbox');`
Just below the startElement, you can write attributes, like so:
`$xmlWriter->writeAttribute('v-text-anchor', $style->getVTextAnchor());`  # unsure about v-text-anchor

Property value `$vTextAnchor` and get/set methods has to be held in `./phpoffice/phpword/src/PhpWord/Style/TextBox.php`

afaict.

**Comment by shapiraman** [comment]
Any update about this?
